server/api/min_resolved_ts.go

```diff
@@ -44,7 +44,7 @@ type minResolvedTS struct {
 // @Success 200 {array} minResolvedTS
 // @Failure 500 {string} string "PD server failed to proceed the request."
 // @Router /min-resolved-ts [get]
-func (h *minResolvedTSHandler) Get(w http.ResponseWriter, r *http.Request) {
+func (h *minResolvedTSHandler) GetMinResolvedTS(w http.ResponseWriter, r *http.Request) {
```

Comment:
can it load minTS from `cluster`?

Reply:
if a store with leader peers doesn't report, the cluster will return zero, and the storage will return the min resolved ts in the last round.